packages/dashboard/app/components/settings/sections/__tests__/settings-default-descriptions.test.tsx

```
@@ -353,6 +360,24 @@ const NOT_SURFACED_ALLOWLIST: Record<string, string> = {
   nested enabled flag rather than a top-level plain description field.
   */
   voiceInput: "nested Voice Input section object; enable toggle owns Default: off for voiceInput.enabled",
+  // Per-project Stash API key override for hard isolation. Not surfaced in Settings —
+  // the PRIMARY stash key lives in the global secrets store and any override is handled
+  // through the secrets path, so this key has no user-editable Settings description field.
+  stashApiKey: "per-project Stash API key override handled via the secrets path, not a Settings UI field",
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)